cmd/export_assets.go

```diff
@@ -29,7 +29,7 @@ var assetsCmd = &cobra.Command{
 		}
 
 		// With seenIDs, the code doesn't export duplicate assets within a single export. Note that across exports, assets may be duplicated
-		seenIDs := map[uint64]bool{}
+		seenIDs := map[int64]bool{}
```

Comment:
Doesn't this need to stay `uint64`?

Reply:
@sydneynotthecity It is because in the `FarmHashAsset` function we change the uint64 to int64, so asset_id is an int64 as was discussed in this [PR](https://github.com/stellar/stellar-etl/pull/183#issuecomment-1478356546)